calico-enterprise/reference/resources/bfdconfig.mdx

```diff
@@ -0,0 +1,63 @@
+---
+description: API for this Calico Enterprise resource.
+---
+
+import Selectors from '@site/calico-enterprise/_includes/content/_selectors.mdx';
+
+# BFD configuration
+
+A BFD configuration resource (`BFDConfiguration`) represents BFD specific configuration options for a selection of one or more nodes.
+
+For `kubectl` commands, the following case-insensitive aliases may be used to specify the resource type on the CLI: `bfdconfiguration.projectcalico.org`, `bfdconfigurations.projectcalico.org` as well as abbreviations such as `bfdconfiguration.p` and `bfdconfigurations.p`.
+
+## Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v3
+kind: BFDConfiguration
+metadata:
+  name: default
+spec:
+  nodeSelector: "all()"
+  interfaces:
+  - matchPattern: "*"
+    minimumRecvInterval: 10ms
+    minimumSendInterval: 100ms
+    idleSendInterval: 1m
+    multiplier: 5
+```
+
+## BFD configuration definition
+
+### Metadata
+
+| Field | Description                                               | Accepted Values                                     | Schema |
+| ----- | --------------------------------------------------------- | --------------------------------------------------- | ------ |
+| name  | Unique name to describe this resource instance. Required. | Alphanumeric string with optional `.`, `_`, or `-`. | string |
+
+### Spec
+
+| Field   | Description   | Accepted Values | Schema  | Default |
+| ------- | ------------- | --------------- | ------- | ------- |
+| nodeSelector | Selects one or more nodes to which this resource applies.  | | string                                                | [selector](#selector) |
+| interfaces   | List of per-interface BFD configuration parameters.        | | List of [BFDInterface](#bfdinterface)                 |                       |
+
+### BFDInterface
+
+| Field   | Description   | Accepted Values | Schema  | Default |
+| ------- | ------------- | --------------- | ------- | ------- |
+| matchPaterrn | Pattern to match one or more interfaces. Supports exact interface names, match on interface prefixes (e.g., “eth*”), or “*” to select all interfaces on the selected node(s). | | string | |
+| minimumRecvInterval | The minimum interval between received BFD packets. Must be a whole number of milliseconds greater than 0. | | string | 10ms |
+| minimumSendInterval | The minimum interval between transmitted BFD packets. Must be a whole number of milliseconds greater than 0. | | string | 100ms |
+| idleSendInterval | The interval between transmitted BFD packets when the BFD peer is idle. Must be a whole number of milliseconds greater than 0. | | string | 1m |
+| multiplier | The number of intervals that must pass without receiving a BFD packet before the peer is considered down. | | string | 5 |
+
+### Selector
+
+<Selectors />
```

Comment:
This may be outside the scope of this PR. I can't see exactly why this selector section is included in some resources but not others. It's generic, not tailored to a particular resource. Has it been ommitted from other pages? Or should it be a standalone page? These are the pages it's used in for CE 3.19:
<img width="299" alt="image" src="https://github.com/user-attachments/assets/9909a2e9-dd28-4c98-a50a-407b80cca60e">
Would be grateful for any thoughts you have here.

Reply:
Selector is a "type" of field (i.e., a string with a particular required structure) that we use across a number (but not all) of our APIs. That's probably why it is only showing up in these documents but not in others.

Personally I don't know whether it's right to have a separate doc or just continue embedding it where needed - the syntax is common across all of these, but selectors are only ever used within the context of one of these resources so I believe that's why we opted for embedding rather than a separate doc.